ds3-cli-certification/src/main/java/com/spectralogic/ds3cli/certification/CertificationUtil.java

```diff
@@ -0,0 +1,168 @@
+
+    public static Path createTempFiles(
+           final String prefix,
+           final int numFiles,
+           final long length) throws IOException {
+        LOG.info("Creating {} files of size {}...", numFiles, length);
+        final Path tempDir = Files.createTempDirectory(prefix);
+        for(int fileNum = 0; fileNum < numFiles; fileNum++) {
+            final File tempFile = new File(tempDir.toString(), prefix + "_" + fileNum);
+            final RandomAccessFile raf = new RandomAccessFile(tempFile, "rw");
+            raf.seek(length);
+            raf.writeBytes("end of RandomAccessFile.");
+            raf.close();
+        }
+        return tempDir;
+    }
+
+    public static String getUserSecretKey(final Ds3Client client, final String username) {
+        try {
+            return client.getUserSpectraS3(new GetUserSpectraS3Request(username)).getSpectraUserResult().getSecretKey();
+        } catch (final Exception e) {
+            LOG.info("Could not get Secret Key fro User {}", username);
+            return "";
+        }
+    }
+
+    public static SpectraUser createUser(final Ds3Client client, final String username) throws IOException {
+        final DelegateCreateUserSpectraS3Response response
+                =  client.delegateCreateUserSpectraS3(new DelegateCreateUserSpectraS3Request(username));
+        return response.getSpectraUserResult();
+    }
+
+    public static void deleteUser(final Ds3Client client, final String username) throws IOException {
+        client.delegateDeleteUserSpectraS3(new DelegateDeleteUserSpectraS3Request(username));
+    }
+
+    public static String createPutJob(final Ds3Client client, final String bucketName, final Priority priority) throws Exception {
+        final Long fileSize = 1024L;
+
+        final Ds3Object ds3Obj = new Ds3Object(bucketName + "_0", fileSize);
+        final PutBulkJobSpectraS3Response putBulkResponse = client.putBulkJobSpectraS3(
+                new PutBulkJobSpectraS3Request(
+                        bucketName,
+                        Lists.newArrayList(ds3Obj)).withPriority(priority));
+        return putBulkResponse.getMasterObjectList().getJobId().toString();
+    }
+
+    public static void deleteJob(final Ds3Client client, final String jobId) {
+        try {
+            final CancelJobSpectraS3Request request = new CancelJobSpectraS3Request(jobId);
+            client.cancelJobSpectraS3(request);
+        } catch (final IOException ex) {
+            LOG.info("Could not delete job Id {}", jobId);
+        }
+    }
+
+    public static void cancelTapeEject(final Ds3Client client, final String barcode) {
+        try {
+            client.cancelEjectTapeSpectraS3(new CancelEjectTapeSpectraS3Request(barcode));
+        } catch (final IOException e) {
+            LOG.info("Failed cancel eject tape, barcode {}", barcode);
```

Comment:
Similar comment as the one above about the format of this error print statement.  This should be updated to:

```java

LOG.error("Failed cancel eject tape, barcode " + barcode, e);
```